elements/winningCondition.js

```diff
@@ -0,0 +1,142 @@
+const html = require('choo/html')
+const sf = require('sheetify')
+const _ = require('lodash')
+const { getGameState, getEntity } = require('../lib/utils/game')
+
+const winPrefix = sf`
+  :host {
+    position: absolute;
+    left: 10%;
+    top: 10%;
+    background-color: #404040;
+    color: white;
+    width: 80%;
+    height: 80%;
+    z-index: 100;
+    display: block;
+    padding: 25px;
+  }
+  
+  :host > .modalContent {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    background-color: #FFFFFF;
+    color: #404040;
+    height: 100%;
+    text-align: center;
+    overflow: scroll;
+  }
+  
+  button {
+    width: 10%;
+  }
+  
+  h1 {
+    margin: 0;
+  }
+  
+  img {
+    width: 50%;
+  }
+`
+
+const goalPrefix = sf`
+  :host {
+    position: absolute;
+    left: 80%;
+    top: 0;
+    height: 20%;
+    width: 20%;
+    background-color: #DDDDDD;
+    color: #404040;
+    overflow: scroll;
+  }
+  
+  :host .modalContent {
+    margin: 25px;
+  }
+`
+
+const winningCondition = (state, send) => {
```

Comment:
I think this component should be split up in two separate components:  goal progress display and dialog